docker-compose.local.yml

```diff
@@ -125,7 +127,7 @@ nginx:
   environment:
     - NGINX_CONFIG_FILE_NAME=nginx_site_http.conf
   ports:
-    # NOTE: The host ports (left of the ":") must match the values in 
+    # NOTE: The host ports (left of the ":") must match the values in
```

Comment:
@yahyazini, you're not wrong to want to remove these trailing spaces, but I'd request you do this kind of cleaning only on lines that you're already editing. For example, if you thought the comment here was inaccurate or needed to be reworded, then by all means remove the sloppy whitespace while you're at it.

When editors automatically make style corrections en masse (e.g. stripping whitespace or reordering all the `import` statements) and these changes are committed, it muddies the `git blame` and clutters up PRs, making them more difficult to review.